src/Effects.js

```diff
@@ -91,24 +91,33 @@ module.exports = class Effects {
         if (this.stars.length) {
           return;
         }
+
         for (let i = 0; i < 75; i++) {
           this.stars.push({
             x: p5.random(0, 400),
-            y: p5.random(0,100),
-            color: "#C0C0C0",
+            y: p5.random(0,200),
+            color: randomColorFromPalette(),
           });
         }
       },
       draw: function () {
-        p5.background("#9370DB");
+        p5.noFill();
+        // Draw a horizontal gradient of the palette colors to the background
+        for (let i = 0; i <= 425; i++) {
+          let c = lerpColorFromPalette(i / 425);
+          p5.stroke(c);
+          p5.line(0, i, 425, i);
```

Comment:
I just saw this is doing a line per row, instead of a [linear gradient](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/createLinearGradient).  I'm a bit worried about the perf of this change.  This is already one of our more CPU-intensive effects.